### PR TITLE
done 

### PR DESCRIPTION
**Pull request by dahmoni1211**

### CODE REVIEW

**Comment by dahmoni1211** [comment]
the activity is done

File: src/App.asm [hunk 1]
```diff
@@ -1,6 +1,55 @@
...
+ 	la $a0,str2
+ 	syscall
+ 
+ 	li $v0,5
+	 syscall
+ 	move   $t2, $v0
+	sw     $t2, score # score rigester
+	add $t1, $t1,$t7
+	lw $t3, top_student
+	bgt $t2,$3,if_body
```
Comment (marwankau) [inline src/App.asm]: you use wrong register `$3`. I think you mean `$t3`. Code must me test before submission and you need to find the issue and fix it

File: src/App.asm [hunk 2]
```diff
@@ -1,6 +1,55 @@
...
+	la     $a0, str1
+	syscall
+	li $v0,5 
+	syscall
+	move $t0,$v0 # count
+	sw $t0,count
+	
+	lw $t1,i
+loop_start:
+ blt $t1,$t0, loopBody
```
Comment (marwankau) [inline src/App.asm]: variables (`i` and `count`) used in while loop condition must be loaded from memory before such as:
```asm
...
lw  $t1, i
lw  $t0. count
blt $t1, t0, loopBody
...
```

File: src/App.asm [hunk 3]
```diff
@@ -1,6 +1,55 @@
...
+ loopBody:
+ 	li $v0,4
+ 	la $a0,str2
+ 	syscall
+ 
+ 	li $v0,5
+	 syscall
+ 	move   $t2, $v0
+	sw     $t2, score # score rigester
+	add $t1, $t1,$t7
```
Comment (marwankau) [inline src/App.asm]: `$t1` value should be stored in `i` each iteration such as:
```asm
...
add $t1, $t1, $t7
sw  $t1, i
...
```

**Pull request closed without merging**